src/objects/DataImportBatch__c.object

```diff
@@ -110,6 +110,19 @@
         <type>LongTextArea</type>
         <visibleLines>3</visibleLines>
     </fields>
+    <fields>
+        <fullName>Batch_Number__c</fullName>
+        <description>The unique, auto-numbered identifier for this batch.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The unique, auto-numbered identifier for this batch.</inlineHelpText>
+        <label>Batch Number</label>
+        <length>30</length>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
```

Comment:
@kenlewis-sfdo would it instead make sense to have this field set as a unique ID that customers could use for an import if they were so inclined?

Reply:
Yes, definitely!  Good catch.  Just updated.